{{cookiecutter.project_slug}}/Makefile

```diff
@@ -99,6 +95,7 @@ test:{#
 	"pytest" -x --log-level=warning tests/unit
 
 
+
```

Comment:
I have a first implementation where the taggin of the docker file is automatically done by reading VERSION in the service folder.
Shall we add there a standard:
```
make push-staging:
  ${DOCKER} tag service:latest ${DOCKER_REGISTRY}/service:staging-latest;\
  ${DOCKER} push ${DOCKER_REGISTRY}/service:staging-latest
make push-release:
  SERVICE_VERSION=$$(cat service/VERSION);\
  ${DOCKER} tag service:latest ${DOCKER_REGISTRY}/service:$$SERVICE_VERSION;\
  ${DOCKER} push ${DOCKER_REGISTRY}/service:$$SERVICE_VERSION;\
```

Reply:
I agree. created issue #8 for that